protobuf 3.2.0 issue

```
libtool: compile:  clang++ -DHAVE_CONFIG_H -I. -I./include -I./include -Wall -Wformat -W -fvisibility-inlines-hidden -D_THREAD_SAFE -isystem /usr/local/Cellar/protobuf/3.2.0/include -DPID_DATA_DIR=\"/usr/local/Cellar/ola/HEAD-5f490bd_1/share/ola/pids\" -g -O2 -std=gnu++98 -D_THREAD_SAFE -c common/rpc/Rpc.pb.cc  -fno-common -DPIC -o common/rpc/.libs/common_libolacommon_la-Rpc.pb.o
libtool: compile:  clang++ -DHAVE_CONFIG_H -I. -I./include -I./include -Wall -Wformat -W -fvisibility-inlines-hidden -D_THREAD_SAFE -isystem /usr/local/Cellar/protobuf/3.2.0/include -DPID_DATA_DIR=\"/usr/local/Cellar/ola/HEAD-5f490bd_1/share/ola/pids\" -g -O2 -std=gnu++98 -D_THREAD_SAFE -c common/rpc/Rpc.pb.cc -o common/rpc/common_libolacommon_la-Rpc.pb.o >/dev/null 2>&1
/bin/sh ./libtool  --tag=CXX   --mode=compile clang++ -DHAVE_CONFIG_H -I.    -I./include -I./include -Wall -Wformat -W -fvisibility-inlines-hidden -D_THREAD_SAFE -isystem /usr/local/Cellar/protobuf/3.2.0/include -g -O2 -std=gnu++98 -D_THREAD_SAFE  -c -o common/protocol/common_protocol_libolaproto_la-Ola.pb.lo `test -f 'common/protocol/Ola.pb.cc' || echo './'`common/protocol/Ola.pb.cc
libtool: compile:  clang++ -DHAVE_CONFIG_H -I. -I./include -I./include -Wall -Wformat -W -fvisibility-inlines-hidden -D_THREAD_SAFE -isystem /usr/local/Cellar/protobuf/3.2.0/include -g -O2 -std=gnu++98 -D_THREAD_SAFE -c common/protocol/Ola.pb.cc  -fno-common -DPIC -o common/protocol/.libs/common_protocol_libolaproto_la-Ola.pb.o
libtool: compile:  clang++ -DHAVE_CONFIG_H -I. -I./include -I./include -Wall -Wformat -W -fvisibility-inlines-hidden -D_THREAD_SAFE -isystem /usr/local/Cellar/protobuf/3.2.0/include -g -O2 -std=gnu++98 -D_THREAD_SAFE -c common/protocol/Ola.pb.cc -o common/protocol/common_protocol_libolaproto_la-Ola.pb.o >/dev/null 2>&1
/bin/sh ./libtool  --tag=CXX   --mode=compile clang++ -DHAVE_CONFIG_H -I.    -I./include -I./include -Wall -Wformat -W -fvisibility-inlines-hidden -D_THREAD_SAFE -isystem /usr/local/Cellar/protobuf/3.2.0/include -g -O2 -std=gnu++98 -D_THREAD_SAFE  -c -o common/protocol/common_protocol_libolaproto_la-OlaService.pb.lo `test -f 'common/protocol/OlaService.pb.cpp' || echo './'`common/protocol/OlaService.pb.cpp
libtool: compile:  clang++ -DHAVE_CONFIG_H -I. -I./include -I./include -Wall -Wformat -W -fvisibility-inlines-hidden -D_THREAD_SAFE -isystem /usr/local/Cellar/protobuf/3.2.0/include -g -O2 -std=gnu++98 -D_THREAD_SAFE -c common/protocol/OlaService.pb.cpp  -fno-common -DPIC -o common/protocol/.libs/common_protocol_libolaproto_la-OlaService.pb.o
common/protocol/OlaService.pb.cpp:27:3: error: use of undeclared identifier 'protobuf_AddDesc_Ola_2eproto'
  protobuf_AddDesc_Ola_2eproto();
  ^
1 error generated.
make[2]: *** [common/protocol/common_protocol_libolaproto_la-OlaService.pb.lo] Error 1
make[1]: *** [all-recursive] Error 1
make: *** [all] Error 2
```

I'm seeing this while trying to upgrade protobuf to 3.2.0 in Homebrew and revision bumping all reverse dependencies. It looks like the Fedora team has hit the same problem for the same reason: https://www.spinics.net/lists/fedora-devel/msg229952.html

--- a/config/ola.m4
+++ b/config/ola.m4
@@ -24,6 +24,9 @@ AC_DEFUN([PROTOBUF_SUPPORT],
 AC_REQUIRE_CPP()
 PKG_CHECK_MODULES(libprotobuf, [protobuf >= $1])
 
+PKG_CHECK_MODULES(libprotobuf2, [protobuf < 3.2], [],
+                  [AC_MSG_ERROR([OLA currently requires protobuf < 3.2, see issue 1192])])
+
 AC_SUBST([libprotobuf_CFLAGS])
 
 AC_ARG_WITH([protoc],

